CHANGES.rst

```diff
@@ -1,5 +1,15 @@
 .. currentmodule:: click
 
+Version 8.1.4
+-------------
+
+Released TBD
+
+-   Do not show an error when attempting to create an option with
+    ``multiple=True, is_flag=True`` if ``flag_value`` is not a ``bool``.
+    :issue:`2292`
```

Comment:
Minor, but may want to rephrase. It wasn't just showing an error, it was raising an exception, breaking a feature.

Reply:
No problem. Would you rather I amend the current commit or submit it as a separate commit?

Reply:
@epruesse Done. Thanks for the suggestion.